src/Session/session.py

```diff
@@ -23,12 +23,8 @@ def _getSavedProfiles(self):
         It will also print a list of profiles the user can load if there are any or
         prompt the user to create a new one. '''
     def _printWelcomeMessage(self):
-        if len(self._profiles):
-            print(WELCOME_MESSAGE, LOAD_CREATE_PROFILE_MESSAGE)
-            print(listSepByTab(self._profiles))
-        else:
-            print(WELCOME_MESSAGE, NO_PROFILES_MESSAGE)
-            print(SUGGESTED_ACTIONS_MESSAGE.format(WITHIN_QUOTES.format(CREATE_PROFILE)))
+        twoPrintOutcomes(self._profiles, [WELCOME_MESSAGE, LOAD_CREATE_PROFILE_MESSAGE], \
```

Comment:
You can just delete `_printWelcomeMessage` and replace the spot it's used with `twoPrintOutcomes` instead of having it call `twoPrintOutcomes`